src/services/landmarks/addLandmarkView.ts

```diff
@@ -0,0 +1,35 @@
+import { isServer } from '../../components/helpers';
+import { fetchJson } from '../fetch';
+import { getShortId, prod } from '../helpers';
+import { Feature } from '../types';
+import { LandmarkView } from './views';
+
+const getViewUrl = (shortId: string) => {
+  const end = `/camera-view?shortId=${shortId}`;
+  if (!isServer()) {
+    return end;
+  }
+  if (prod) {
+    return `https://osmapp.org${end}`;
+  }
+  // TODO: Find the *right* url also for preview deployments
+  return `http://localhost:3000${end}`;
```

Comment:
This currently blocks preview deployments but i don't know how to get the correct url